tests/page-objects/contacts/contacts.po.js

```diff
@@ -15,16 +14,19 @@ const contactName = element(by.css('.heading-content'));
 const rows = element.all(by.className('content-row'));
 const dateOfBirthField = element(by.css('[placeholder="yyyy-mm-dd"]'));
 const contactSexField = element(by.css('[data-name="/data/contact/sex"][value="female"]'));
+const peopleRows = element.all(by.repeater('group in contactsContentCtrl.selectedContact.children'));
+const deleteContact = $('[ng-click="deleteDoc(actionBarCtrl.selectedContactDoc)"]');
```

Comment:
Again this feels brittle to me. In general I would stay well away from the `ng` attributes for selectors. Firstly scope it to the action bar so you don't try and click the wrong button, then either check the icon or the label text to get the exact button you want, eg: `.action-container .detail-actions:not(.ng-hide) .fa-trash-o`